static/app/views/releases/detail/overview/issues.tsx

```diff
@@ -270,6 +423,24 @@ const OpenInButtonBar = styled(ButtonBar)`
   }
 `;
 
+const StyledButtonBar = styled(ButtonBar)`
+  grid-template-columns: repeat(4, 1fr);
+  ${ButtonLabel} {
+    white-space: nowrap;
+    grid-gap: ${space(0.5)};
+    span:last-child {
+      color: ${p => p.theme.gray400};
```

Comment:
how does it look like on dark mode? should we be using an alias here?

Reply:
@priscilawebdev here's what it looks like in Dark Mode 
<img width="842" alt="Screen Shot 2021-06-25 at 12 46 21 AM" src="https://user-images.githubusercontent.com/20312973/123390942-52e87c80-d550-11eb-9900-5e526ee5836b.png">

Would you mind explaining what an alias means here? 😅

Reply:
alias are properties present in our theme that have defined colors to work best in light and dark mode.

https://github.com/getsentry/sentry/blob/e697816db49171869611e962aa09e756729af0f5/static/app/utils/theme.tsx#L54

https://github.com/getsentry/sentry/blob/e697816db49171869611e962aa09e756729af0f5/static/app/utils/theme.tsx#L600

Reply:
I think the `progressBackground` alias has the colors you need, although the name is not ideal :/

Reply:
dark mode looks good 🙂